etc/env/example.env

```diff
@@ -9,14 +9,6 @@ CANDIG_AUTH_MODULES=keycloak tyk opa vault arbiter
 CANDIG_DOMAIN=docker.localhost
 CANDIG_AUTH_DOMAIN=auth.docker.localhost
 
-# candig server
```

Comment:
why deprecate?

Reply:
not deprecated, this was a duplicate and is mentioned again below on this line https://github.com/CanDIG/CanDIGv2/blob/amanjeev/dig-633-authentication/etc/env/example.env#L193-L198